tracing/src/main/java/org/axonframework/extensions/tracing/TracingCommandGateway.java

```diff
@@ -141,7 +141,7 @@ protected TracingCommandGateway(Builder builder) {
         FutureCallback<Object, R> futureCallback = new FutureCallback<>();
 
         CommandMessage<?> cmd = GenericCommandMessage.asCommandMessage(command);
-        sendWithSpan(tracer, "sendCommandMessageAndWait", cmd, (tracer, parentSpan, childSpan) -> {
+        sendWithSpan(tracer, "sendAndWait"+ SpanUtils.messageName(cmd), cmd, (tracer, parentSpan, childSpan) -> {
```

Comment:
As mentioned in a comment on the `OpenTraceHandlerInterceptor`, I think we should add an underscore between `sendAndWait` and the resolved message name.

Reply:
check